WordPress/Classes/ViewRelated/Reader/ReaderCommentCell.swift

```diff
@@ -0,0 +1,217 @@
+import UIKit
+import WordPressShared
+import Gridicons
+
+@objc protocol ReaderCommentCellDelegate: WPRichContentViewDelegate
+{
+    func cell(cell: ReaderCommentCell, didTapAuthor comment: Comment)
+    func cell(cell: ReaderCommentCell, didTapLike comment: Comment)
+    func cell(cell: ReaderCommentCell, didTapReply comment: Comment)
+}
+
+class ReaderCommentCell : UITableViewCell
+{
+    var enableLoggedInFeatures = false
+
+    @IBOutlet var avatarImageView: UIImageView!
+    @IBOutlet var authorButton: UIButton!
+    @IBOutlet var timeLabel: UILabel!
+    @IBOutlet var textView: WPRichContentView!
+    @IBOutlet var replyButton: UIButton!
+    @IBOutlet var likeButton: UIButton!
+    @IBOutlet var actionBar: UIStackView!
+    @IBOutlet var leadingContentConstraint: NSLayoutConstraint!
+
+    weak var delegate: ReaderCommentCellDelegate? {
+        didSet {
+            textView.delegate = delegate
+        }
+    }
+
+    var comment: Comment?
+
+    var showReply: Bool {
+        get {
+            if let comment = comment, let post = comment.post as? ReaderPost {
+                return post.commentsOpen && enableLoggedInFeatures
+            }
+            return false
+        }
+    }
+
+    override var indentationLevel: Int {
+        didSet {
+            leadingContentConstraint.constant = CGFloat(indentationLevel) * indentationWidth
+        }
+    }
+
+    override var indentationWidth: CGFloat {
+        didSet {
+            leadingContentConstraint.constant = CGFloat(indentationLevel) * indentationWidth
+        }
+    }
+
+    // MARK: - Lifecycle Methods
+
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        setupReplyButton()
+        setupLikeButton()
+        applyStyles()
+    }
+
+
+    // MARK: = Setup
+
+
+    func applyStyles() {
+        WPStyleGuide.applyReaderCardSiteButtonStyle(authorButton)
+        WPStyleGuide.applyReaderCardBylineLabelStyle(timeLabel)
+
+        authorButton.titleLabel?.lineBreakMode = .ByTruncatingTail
+        textView.textContainerInset = UIEdgeInsets(top: 2, left: -4, bottom: -16, right: 0)
+    }
+
+
+    func setupReplyButton() {
+        let icon = Gridicon.iconOfType(.Reply, withSize: CGSize(width: 20, height: 20))
+        let tintedIcon = icon.imageWithTintColor(WPStyleGuide.grey())
+        let highlightedIcon = icon.imageWithTintColor(WPStyleGuide.lightBlue())
+
+        replyButton.setImage(tintedIcon, forState: .Normal)
+        replyButton.setImage(highlightedIcon, forState: .Highlighted)
+
+        let title = NSLocalizedString("Reply", comment: "Title of the reply button.")
```

Comment:
Would we want the localized description to read with something like `"Reader comments: Title of the reply button."`?